server/index.js

```diff
@@ -15,7 +15,7 @@ const PORT = process.env.PORT || process.env.LOCAL_PORT;
 const server = express();
 
 server.use(bodyParser.json());
-server.use(cors());
+server.use(cors({credentials: true, origin: process.env.CLIENT_URL }));
```

Comment:
When loading config vars it is often helpful to have a sensible default so that your app will start if the config file can't be read or the value is missing. 

something like this:

`const client_url = process.env.CLIENT_URL || 'http://localhost:3000'
server.use(cors({credentials: true, origin: client_url }));
`